cli/builtin.h

```diff
@@ -43,6 +43,7 @@ COMMAND_LIST(
 	ENTRY("evcntr-show", "Show an event counters setup info", evcntr_show)
 	ENTRY("evcntr-del", "Deconfigure an event counter", evcntr_del)
 	ENTRY("evcntr-wait", "Wait for an event counter to exceed its threshold", evcntr_wait)
+	ENTRY("mrpc-ses", "Send an MRPC SES command to the firmware", mrpc_ses)
 );
```

Comment:
Every command is an mrpc command and none of them have an mrpc- prefix. This should just be 'ses' and save the user some typing.